org_status/org_status.py

```diff
@@ -137,4 +169,12 @@ def main():
 
         org_host = Host(token, org, verbose=args.verbose)
         org_status = aggregate_org_status(org_host, threads=args.threads)
```

Comment:
```suggestion
        if args.export_orgs:
            export_data = encode_repo_list(org_status, args.format, styled)
            export_file = path.abspath(args.export_orgs)
              
            if export_data is not None:
                write_data_to_file(export_data, export_file, styled, verbose)
            
            return
              
        org_status = aggregate_org_status(org_host, threads=args.threads)
```

Reply:
We should something like this, this way we can prevent it from checking the repo statuses before dumping the repo list.